packages/flutter_tools/lib/src/flutter_manifest.dart

```diff
@@ -703,3 +691,88 @@ void _validateFonts(YamlList fonts, List<String> errors) {
     }
   }
 }
+
+/// Represents an entry under the `assets` section of a pubspec.
+@immutable
+class AssetsEntry {
+  const AssetsEntry({
+    required this.uri,
+    this.flavor,
+  });
+
+  final Uri uri;
+  final String? flavor;
+
+  static const String _pathKey = 'path';
```

Comment:
constants should have a `k` prefix: `static const String _kPathKey = 'path'`

Reply:
IIRC our style guide only applies this rule to global constants ([wiki](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k)). Do you believe all constants should be prefixed?

Reply:
I generally do, but perhaps we don't follow this as strictly as I thought.